src/modules/Gapi.ts

```diff
@@ -57,19 +57,74 @@ class GoogleApi {
         }
     }
 
-    searchYoutube = async (searchTerm: string) => {
+    searchYoutubeList = async (searchTerm: string) => {
         try {
             const response = await this.gapi.client.youtube.search.list({
                 "part": [
                     "snippet"
                 ],
                 "q": searchTerm
             })
+            let videoIds: string[] = [];
+
+            response.result.items.forEach((item: any) => {
+                videoIds.push(item.id.videoId)
+            });
+            return videoIds;  
+        } catch(err) {
+            console.log('searchYoutubeList: ERROR ', err)
```

Comment:
cleanup

Reply:
Is there any other way for error logs?

Reply:
We don't debug in prod.
We have to do appropriate error handling

Reply:
```
async searchYoutubeList(searchTerm: string): Promise<string[]> {
        try {
            const response = await this.gapi.client.youtube.search.list({
                "part": [
                    "snippet"
                ],
                "q": searchTerm
            })
            let videoIds: string[] = [];

            response.result.items.forEach((item: any) => {
                videoIds.push(item.id.videoId)
            });

            return videoIds;  
        } catch(err) {
            Promise.reject(err); // throws error: Function lacks ending return statement and return type does not include 'undefined'.
        }
    }
```
   
 Can you please suggest a way to handle the error here?

Reply:
>We don't debug in prod.

Was asking for a suggestion for debugging in dev.